extras/mmu/mmu.py

```diff
@@ -2456,14 +2459,14 @@ def cmd_MMU_CALIBRATE_BOWDEN(self, gcmd):
 
         try:
             with self.wrap_sync_gear_to_extruder():
-                self.calibrating = True
-                if manual:
-                    self._calibrate_bowden_length_manual(approx_bowden_length, save)
-                else:
-                    # Automatic method with encoder
-                    self._reset_ttg_map() # To force tool = gate
-                    self._unload_tool()
-                    with self._require_encoder():
```

Comment:
We had another check here that was requiring the encoder for the auto calibration, but removed that, the condition check earlier in the method should be sufficient and covers encoder or extruder entry.